src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -67,8 +61,12 @@ internal OtlpExporterOptions(
         BatchExportActivityProcessorOptions defaultBatchOptions)
     {
         Debug.Assert(defaultBatchOptions != null, "defaultBatchOptions was null");
+        var finalConfiguration = new ConfigurationBuilder()
+            .AddEnvironmentVariables(OtlpSpecConfigDefinitions.EnvVarPrefix)
```

Comment:
This _might_ have unintended consequences if only environment variables with this prefix are consumed now.

Reply:
I _think_ it makes sense for any specified `IConfiguration` values to override the environment variables.